packages/angular/src/schematics/application/application.spec.ts

```diff
@@ -113,7 +113,7 @@ describe('app', () => {
       let myAppPrefix = workspaceJson.projects['my-app'].prefix;
 
       expect(myAppPrefix).toEqual('proj');
-      expect(appE2eSpec).toContain('Welcome to my-app!');
+      expect(appE2eSpec).toContain('my-app app is running!');
```

Comment:
why did this changed?

Reply:
This should be reverted. I'll have to update the e2e spec also to replace the text